client/components/scrollable-horizontal-navigation/styles.scss

```diff
@@ -0,0 +1,101 @@
+.is-section-reader .navigation-header.discover-stream-header {
```

Comment:
All the styles have been extracted from `client/reader/discover/discover-navigation.scss`, replacing `.discover-stream-navigation` with `.scrollable-horizontal-navigation`.